bundle/jsky.app.ot/src/main/scala/jsky/app/ot/ags/BagsManager.scala

```diff
@@ -108,18 +106,25 @@ object BagsManager {
     }
   }
 
-  private val NumThreads = math.max(1, Runtime.getRuntime.availableProcessors()-1)
+  private val NumThreads = math.min(1, Runtime.getRuntime.availableProcessors())
```

Comment:
There might be a problem with this PR?  It seems to be undoing work that you did previously?